examples/todo-intermediate/src/datasources/todo-list-db.datasource.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "TodoListDb",
+  "connector": "memory",
+  "localStorage": "",
+  "file": ""
```

Comment:
I'll probably end up persisting the memory db with some examples here

Reply:
Is there any particular reason for using a different datasource for Todo and TodoList models? IMO, most applications will use the same datasource (think of a MySQL database or MongoDB collection) to store data of all models.

I am proposing to keep the `todo-intermediate` example simple and use the same `db` datasource for both `Todo` and `TodoList`.